challenges/fifo_animal_shelter/fifo_animal_shelter.py

```diff
@@ -0,0 +1,68 @@
+class AnimalShelter():
+
+    def __init__(self):
+        self.headCat = None
+        self.headDog = None
+        self.tailCat = None
+        self.tailDog = None
+        self.animalAge = 0
+
+    """ enqueues """
+    def enqueue(self, animalName, animalType):
+        """ adds animals to the shelter """
+        self.animalAge += 1
+        newAnimal = Node(animalName, animalType)
+        newAnimal.timestamp = self.animalAge
+        if animalType == 'cat':
+
+            if not self.headCat:
+                self.headCat = newAnimal
+
+        if self.tailCat:
+            self.tailCat.pointer = newAnimal
+            self.tailCat = newAnimal
+
+        elif animalType == 'dog':
+
+            if not self.headDog:
+                self.headDog = newAnimal
+
+        if self.tailDog:
+            self.tailDog.pointer = newAnimal
+            self.tailDog = newAnimal
+
+    """ Dequeues """
+    def dequeuePrefDog(self):
```

Comment:
These methods are correct, however the spec calls for a method named `.dequeue()` that has the `pref` as an parameter. Just make these methods private and add a `.dequeue(pref)` method that calls them based on the `pref`.